### ISSUE TITLE
Windows 10 - Last Build/Patch 10.0.19045.2251 - Notifications N/A

### ISSUE THREAD

**Issue by verbal666** [RandomEngy/VidCoder#1083]
### Problem Description

Under Windows 10.0.19045.2251 64, Notifications for VidCoder are greyed out and not changeable.

![Cattura](https://user-images.githubusercontent.com/12701960/207509964-0737083e-e599-4c57-a6d1-a22e7bba70d6.JPG)

### What version of VidCoder are you running?

Both 7.15 and 8.12b

### Encode Log

```text
N/A
```

**Comment by RandomEngy** [comment]
On Windows 11 it works fine for me:

![image](https://user-images.githubusercontent.com/9450861/207931453-1dc45c77-d55e-485f-bf3a-bc59c9a9ff1a.png)

I'm not sure what might be going on. Are notifications working for other apps?

**Comment by RandomEngy** [comment]
That's probably it. I do know that the notifications API can require a shortcut that the installer adds.

**Comment by verbal666** [comment]
Yes. It's the portable version. I tried on 2 W10/11 VMs, and the issue is there only on W10,

W10 VM
![Cattura](https://user-images.githubusercontent.com/12701960/208034238-17faae79-1ab9-4e23-8b71-5ecaee7be773.PNG)

W11 VM
[no VidCoder detected in apps]

No problem. Just to know the Portable Version on W10 does not create an interactive Notification App 🤷‍♀️
